internal/pkg/print/markdown/table/table.go

```diff
@@ -64,10 +64,10 @@ const (
 		{{ if not .Module.Outputs }}
 			No output.
 		{{ else }}
-			| Name | Description |
-			|------|-------------|
+			| Name | Description |{{ if $.Settings.OutputValues }} Value |{{ end }}
+			|------|-------------|{{ if $.Settings.OutputValues }}-------|{{ end }}
 			{{- range .Module.Outputs }}
-				| {{ name .Name }} | {{ tostring .Description | sanitizeTbl }} |
+				| {{ name .Name }} | {{ tostring .Description | sanitizeTbl }} |{{ if $.Settings.OutputValues }} {{ .Value | sanitizeInterface | sanitizeTbl }} | {{ end }}
```

Comment:
There's an extra space right before the last `| {{ end }}` which should be removed (that's a markdown lint warning)

Reply:
Okay, fixing that now. Btw sorry if my PR has become a bit of a clusterf. I'm not super accustomed to squashing/resolving merge conflicts :stuck_out_tongue: Will make changes for this shortly!

Reply:
No, it's completely find, not squashing commits and PR of this size is actually a better thing, makes with code review. And I'll squash merge the PR anyways, so no worries!